tests/base/models/repository.py

```diff
@@ -130,20 +131,21 @@ def test_repo_url_for_path_no_match(repo_git0, locale0, settings):
 
 @pytest.mark.django_db
 def test_repo_pull(repo_git0):
-
-    with patch('pontoon.base.models.update_from_vcs') as m_update_from_vcs:
-        with patch('pontoon.base.models.get_revision') as m_get_revision:
-            repo_git0.url = 'https://example.com'
-            m_get_revision.return_value = 'asdf'
-            assert (
-                repo_git0.pull()
-                == {'single_locale': 'asdf'})
-            assert (
-                m_update_from_vcs.call_args[0]
-                == ('git',
-                    'https://example.com',
-                    repo_git0.checkout_path,
-                    u''))
+    _patch_ctx = nested(
+        patch('pontoon.base.models.update_from_vcs'),
+        patch('pontoon.base.models.get_revision'))
+    with _patch_ctx as (m_update_from_vcs, m_get_revision):
```

Comment:
Hah, interesting! Just out of curiosity, is it still possible to use the `@patch` decorator with pytest fixtures, or do they conflict?

Reply:
they conflict - you can kinda workaround, but its ugly